src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -137,8 +137,9 @@ class connectionStrings extends viewModelBase {
         }    
     }   
 
-    isConnectionStringInUse(connectionStringName: string) :boolean {
-        return _.includes(Object.keys(this.connectionStringsTasksInfo), connectionStringName);       
+    isConnectionStringInUse(connectionStringName: string, task: string): boolean {
+        return _.includes(Object.keys(this.connectionStringsTasksInfo), connectionStringName)
+            && (typeof this.connectionStringsTasksInfo[connectionStringName].find(x => x.TaskType === task) === 'object');
```

Comment:
I think typeof is not needed here. 

```suggestion
            && this.connectionStringsTasksInfo[connectionStringName].find(x => x.TaskType === task);
```

Reply:
done